web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -129,7 +129,7 @@
                     {{ k }}
                   </v-subheader>
                   <v-list-item
-                    :key="k"
+                    :key="`${k}-item`"
```

Comment:
I'm not sure this helps anything. The key is still solely determined by the position of `item` within `extraFields`, so any changes to absolute positions will confuse the re-rendering.

Reply:
`extraFields` is an object, so shouldn't the key (`k`) for each item be sufficiently unique?